recipes/flint/meta.yaml

```diff
@@ -0,0 +1,40 @@
+S{% set version = "2.5.3.dev0" %}
+{% set commit = "6e1c767c29b5d1c0e94d33512181ad63e73f5fc7" %}
```

Comment:
I've use the git version because all releases are GPL 2 and the development version is LGPL 2